pkg/instrumentors/bpf/net/http/client/bpf/probe.bpf.c

```diff
@@ -54,8 +53,10 @@ static __always_inline long inject_header(void* headers_ptr, struct span_context
 
     // Currently only maps with less than 8 keys are supported for injection
     if (map_keyvalue_count >= 8) {
+        bpf_printk("Map size is bigger than 8, skipping context propagation");
```

Comment:
My understanding is `bpf_printk` has non-negligible performance impact. Maybe we should review where we use them, and have some sort of flag to enable / disable? There are a few 

I'll create a ticket to track from this comment.